lib/beaker/tasks/quick_start.rb

```diff
@@ -0,0 +1,49 @@
+require 'beaker-hostgenerator'
+
+namespace :beaker_quickstart do
+
+  desc 'Generate Default Beaker Host Config File'
+  task :gen_hosts do
```

Comment:
Ok, I was thinking that since they are all part of the quickstart workflow, it would make sense to have them in the same namespace? Maybe I should create a secondary level of namespaces to differentiate the generation tasks from the run task?